src/blocks/load.rs

```diff
@@ -136,10 +148,23 @@ impl Block for Load {
                           "{5m}" => split[1],
                           "{15m}" => split[2]);
 
-        let used_perc = values["{1m}"]
-            .parse::<f32>()
-            .block_error("load", "failed to parse float percentage")?
-            / self.logical_cores as f32;
+        /* Parse all the loadavg values. The block is
+         * visible if they are all greater than their min
+         * in self.minimum_visible */
+        let mut parsed = [0.0; 3];
+        for (e, v) in parsed.iter_mut().zip(split.iter()) {
+            *e = v
+                .parse::<f32>()
+                .block_error("load", "failed to parse load value as float")?;
+        }
+
+        self.visible = self
+            .minimum_visible
+            .iter()
+            .zip(parsed.iter())
+            .all(|(min, v)| v > min);
```

Comment:
It would be more intuitive to have `>=` IMHO. Consider minimum_visible 1.0: should 1.00 be displayed?
```suggestion
            .all(|(min, v)| v >= min);
```